### PR TITLE
fix spectator mode chunk flashing (1.19.0)

### PR DESCRIPTION
**Pull request by Lunarelly**
## Introduction
Chunk flashing in new 1.19.0 update

**Note: wrong verison in commit's name**

### CODE REVIEW

File: src/player/Player.php [hunk 1]
```diff
@@ -2342,6 +2342,7 @@ protected function syncNetworkData(EntityMetadataCollection $properties) : void{
 
 		$properties->setPlayerFlag(PlayerMetadataFlags::SLEEP, $this->sleeping !== null);
 		$properties->setBlockPos(EntityMetadataProperties::PLAYER_BED_POSITION, $this->sleeping !== null ? BlockPosition::fromVector3($this->sleeping) : new BlockPosition(0, 0, 0));
+		$properties->setGenericFlag(EntityMetadataFlags::HAS_COLLISION, $this->getGamemode() !== GameMode::SPECTATOR());
```
Comment (alvin0319) [inline src/player/Player.php]:
```suggestion
		$properties->setGenericFlag(EntityMetadataFlags::HAS_COLLISION, !$this->getGamemode()->equals(GameMode::SPECTATOR());
```

**Comment by alvin0319** [comment]
Oops, I made mistake in my suggestion, you should close one more `)`

**Comment by Lunarelly** [comment]
fixed

**Comment by dktapps** [comment]
This doesn't fix anything.

**Comment by Lunarelly** [comment]
wdym doesnt fix anything? this fixes flashing chunks. you can try it by yourself in spectator mode

**Comment by dktapps** [comment]
I did. That's how I know it doesn't fix anything.

**Comment by Lunarelly** [comment]
it worked for me. idk why it does not work for you

**Pull request closed without merging**